### PR TITLE
Fixed after merge

### PR DESCRIPTION
**Pull request by petemoore**
So my last PR hit a merge conflict, since I hadn't pulled the latest commit when I applied the fix (whoops). Then after  resolving the merge conflict, I hadn't spotted that it impacted a test, so this time I've based my fix on the latest commit, rather than an earlier one, and everything should now be working properly! :cactus: 

Thanks!

CC: @djmitche

### CODE REVIEW

**Comment by petemoore** [comment]
@jonasfj By the way, how can I deploy this?

File: server.js [hunk 1]
```diff
@@ -41,14 +41,10 @@ var createServer = function(options) {
...
-      if (!_.endsWith(q.name.toLowerCase(), options.domain)) {
-        return;
-      }
-
       // Find labels
-      var labels = q.name.split('.');
-      if (labels.length !== 3) {
+      var labels = q.name.toLowerCase().split('.');
+      // Make sure it ends with the domain configured
+      if (labels.slice(1).join('.') !== options.domain.toLowerCase()) {
```
Comment (jonasfj) [inline server.js:47]: toLowerCase is on the wrong side... Yes, some DNS servers changes cases for fun...

Actually I think they use the case bit for error correction, but I haven't had time to dive into that...

**Comment by jonasfj** [comment]
it's deployed on docker-cloud... I don't know anymore either, it ought to be push and then docker hub rebuilds and cloud redeploys... But since hub doesn't have the same github link features/support anymore... And it doesn't work with our org because of security restrictions...

I'm going to go with.... probably we have to build the docker image, push to hub and change the version of the docker image used in cloud.docker.com

File: server.js (hunk 1 above)
Comment (petemoore) [inline server.js:47]: The `.toLowerCase()` is on _both_ sides to be safe. I did this to allow for it being in uppercase in the server config too.

Comment (jonasfj) [inline server.js:47]: oh, missed it...

**Comment by jonasfj** [comment]
# `r+`

**Pull request merged**